packages/financial-templates-lib/src/price-feed/ForexDailyPriceFeed.js

```diff
@@ -205,8 +205,11 @@ class ForexDailyPriceFeed extends PriceFeedInterface {
     const newHistoricalPricePeriods = Object.keys(historyResponse.rates)
       .map(dateString => ({
         // Output data should be a list of objects with only the open and close times and prices.
-        openTime: this._dateTimeToSecond(dateString) - 24 * 3600,
-        closeTime: this._dateTimeToSecond(dateString),
```

Comment:
@mrice32 this was the error before: `openTime` should be beginning of the day, not beginning of the day prior. This is because `closeTime` was getting set to the beginning of the desired day.